app/store/reducers/appState/login.js

```diff
@@ -1,17 +1,17 @@
 // constants
-import { GET_ACCESS_TOKEN } from '../../constants/actionTypes';
+import { RETRIEVE_ACCESS_TOKEN } from '../../constants/actionTypes';
 
 const initialState = {
   accessToken: '',
 };
 
 export default function reducer(state = initialState, action) {
   switch (action.type) {
-    case GET_ACCESS_TOKEN:
-
-      return Object.assign({}, state, {
+    case RETRIEVE_ACCESS_TOKEN:
```

Comment:
GET_ACCESS_TOKEN -> modify state thats suspicious no? :thinking:
should'nt it simply be set